packages/addon/src/addon.ts

```
@@ -732,30 +625,27 @@ builder.defineStreamHandler(
               return 0;
 
             case 'date_first':
-              // We don't sort heavily by date locally - the API already did that
-              // Just do minimal local sorting for quality/language
-              if (aScore !== bScore) {
-                return bScore - aScore;
+              // Sort by date first using the file's date info
+              const aDate = aFile?.['5'] ? new Date(aFile['5']).getTime() : 0;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)